Update dependency to resolve test failure

Build and test are failing due bad dependency.

```
Sending build context to Docker daemon  4.899MB
Step 1/4 : FROM golang:1.11
 ---> 943a5a7ecac8
Step 2/4 : WORKDIR /go/src/github.com/deislab/oras
 ---> Running in 7b6a065e4570
Removing intermediate container 7b6a065e4570
 ---> e1e3ba161997
Step 3/4 : COPY . .
 ---> 01a91c7466c7
Step 4/4 : RUN make test
 ---> Running in 8b87e155213d
go get -u github.com/golang/dep/cmd/dep
/go/bin/dep ensure -v --vendor-only
(1/59) Wrote github.com/gorilla/mux@v1.7.0
(2/59) Wrote github.com/morikuni/aec@master
(3/59) Wrote github.com/prometheus/client_model@master
(4/59) Wrote github.com/opencontainers/go-digest@v1.0.0-rc1
(5/59) Wrote github.com/prometheus/common@v0.2.0
(6/59) Wrote github.com/pmezard/go-difflib@v1.0.0
(7/59) Wrote github.com/opencontainers/image-spec@v1.0.1
(8/59) Wrote github.com/prometheus/procfs@master
(9/59) Wrote github.com/phayes/freeport@1.0.2
(10/59) Wrote github.com/inconshreveable/mousetrap@v1.0
(11/59) Wrote github.com/kardianos/osext@master
(12/59) Wrote github.com/konsorten/go-windows-terminal-sequences@v1.0.1
(13/59) Wrote github.com/matttproud/golang_protobuf_extensions@v1.0.1
(14/59) Wrote github.com/pkg/errors@v0.8.1
(15/59) Wrote github.com/miekg/dns@0d29b283ac0f967dd3a02739bf26a22702210d7a
(16/59) Wrote github.com/prometheus/client_golang@v0.9.2
(17/59) Wrote rsc.io/letsencrypt (from https://github.com/dmcgowan/letsencrypt.git)@master
(18/59) Wrote github.com/sirupsen/logrus@v1.3.0
(19/59) Wrote github.com/opencontainers/runc@v0.1.1
(20/59) Wrote github.com/spf13/cobra@v0.0.3
(21/59) Wrote github.com/spf13/pflag@v1.0.3
(22/59) Wrote github.com/stretchr/testify@v1.3.0
(23/59) Wrote github.com/yvasiyarov/go-metrics@master
(24/59) Wrote github.com/yvasiyarov/gorelic@v0.0.6
(25/59) Wrote github.com/yvasiyarov/newrelic_platform_go@b21fdbd4370f3717f3bbd2bf41c223bc273068e6
(26/59) Wrote golang.org/x/net@master
(27/59) Wrote golang.org/x/sync@master
(28/59) Wrote golang.org/x/crypto@master
(29/59) Wrote golang.org/x/time@master
(30/59) Wrote golang.org/x/sys@master
(31/59) Wrote golang.org/x/text@v0.3.0
(32/59) Wrote github.com/xenolf/lego@a9d8cec0e6563575e5868a005359ac97911b5985
(33/59) Wrote gopkg.in/square/go-jose.v1@v1.1.2
(34/59) Wrote gopkg.in/yaml.v2@v2.2.2
(35/59) Wrote github.com/Azure/go-ansiterm@master
(36/59) Wrote google.golang.org/grpc@v1.18.0
(37/59) Wrote github.com/Shopify/logrus-bugsnag@master
(38/59) Wrote github.com/docker/distribution@v2.7.1
(39/59) Wrote github.com/Microsoft/go-winio@v0.4.11
(40/59) Wrote github.com/bshuster-repo/logrus-logstash-hook@v0.4.1
(41/59) Wrote github.com/beorn7/perks@master
(42/59) Wrote github.com/bugsnag/panicwrap@v1.2.0
(43/59) Wrote google.golang.org/genproto@master
(44/59) Failed to write github.com/bugsnag/bugsnag-go@109a9d63f2b57c4623f4912b256bac4a7a63517d
(45/59) Failed to write github.com/containerd/containerd@v1.2.2
(46/59) Failed to write github.com/containerd/continuity@master
(47/59) Failed to write github.com/davecgh/go-spew@v1.1.1
(48/59) Failed to write github.com/docker/cli@v18.09.1
(49/59) Failed to write github.com/docker/libtrust@master
(50/59) Failed to write github.com/docker/docker@master
(51/59) Failed to write github.com/docker/docker-credential-helpers@v0.6.1
(52/59) Failed to write github.com/docker/go-metrics@master
(53/59) Failed to write github.com/docker/go-connections@v0.4.0
(54/59) Failed to write github.com/gogo/protobuf@v1.2.0
(55/59) Failed to write github.com/docker/go-units@v0.3.3
(56/59) Failed to write github.com/garyburd/redigo@v1.6.0
(57/59) Failed to write github.com/gofrs/uuid@v3.2.0
(58/59) Failed to write github.com/golang/protobuf@v1.2.0
(59/59) Failed to write github.com/gorilla/handlers@v1.4.0
grouped write of manifest, lock and vendor: error while writing out vendor tree: failed to write dep tree: failed to export github.com/bugsnag/bugsnag-go: fatal: failed to unpack tree object 109a9d63f2b57c4623f4912b256bac4a7a63517d
: exit status 128
make: *** [vendor] Error 1
Makefile:40: recipe for target 'vendor' failed
The command '/bin/sh -c make test' returned a non-zero code: 2
```

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -45,7 +45,7 @@
   version = "v0.4.1"
 
 [[projects]]
-  digest = "1:7643865a2cdb66fb14d8361611a75b370a99211ac44983505b3eecb4a6a7a882"
+  digest = "1:0e3ee0c357dba5073fa9ac183b3380fc3c500e8eca5d3b951b00a6bb8415664e"
   name = "github.com/bugsnag/bugsnag-go"
   packages = [
     ".",
@@ -55,7 +55,8 @@
     "sessions",
   ]
   pruneopts = "UT"
-  revision = "109a9d63f2b57c4623f4912b256bac4a7a63517d"
+  revision = "d5b63da7e1f93d83e3980e5ab2bf33179bad7b2a"
+  version = "v1.5.0"
 
 [[projects]]
   digest = "1:3049c43c6d1cfaa347acd27d6342187f8f38d9f416bbba7b02b43f82848302d2"
@@ -66,7 +67,7 @@
   version = "v1.2.0"
 
 [[projects]]
-  digest = "1:1872596d45cb52913fcdea90d468f3e57435959e9c0d99ccb316ee76de341313"
+  digest = "1:37f8940c4d3c41536ea882b1ca3498e403c04892dfc34bd0d670ed9eafccda9a"
   name = "github.com/containerd/containerd"
   packages = [
     "content",
@@ -79,8 +80,8 @@
     "remotes/docker",
   ]
   pruneopts = "UT"
-  revision = "9754871865f7fe2f4e74d43e2fc7ccd237edcbce"
-  version = "v1.2.2"
+  revision = "894b81a4b802e4eb2a91d1ce216b8817763c29fb"
+  version = "v1.2.6"
 
 [[projects]]
   branch = "master"

--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -4,7 +4,7 @@
 
 [[constraint]]
   name = "github.com/containerd/containerd"
-  version = "1.2.1"
+  version = "1.2.6"
 
 [[constraint]]
   name = "github.com/opencontainers/go-digest"
@@ -17,25 +17,9 @@
 # Test dependency
 [[constraint]]
   name = "github.com/docker/distribution"
-  version = "^2.7.1"
+  version = "v2.7.1"
 
 # These overrides below necessary for using docker/distribution as a test dependency
-[[override]]
-  name = "github.com/yvasiyarov/newrelic_platform_go"
-  revision = "b21fdbd4370f3717f3bbd2bf41c223bc273068e6"
-
-[[override]]
-  name = "github.com/miekg/dns"
-  revision = "0d29b283ac0f967dd3a02739bf26a22702210d7a"
-
-[[override]]
-  name = "github.com/xenolf/lego"
-  revision = "a9d8cec0e6563575e5868a005359ac97911b5985"
-
-[[override]]
-  name = "github.com/bugsnag/bugsnag-go"
-  revision = "109a9d63f2b57c4623f4912b256bac4a7a63517d"
-
 [[override]]
   name = "rsc.io/letsencrypt"
   branch = "master"

